app/components/ItunesCard/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Tooltip, Button } from 'antd';
+import styled from 'styled-components';
+import { T } from '../T/index';
+import { LinkOutlined } from '@ant-design/icons';
```

Comment:
import order
1. 3rd party packages
2. aliases
3. relative imports

Reply:
Changed the order